test/interpreter/kernel/exec/numeric-instructions.js

```diff
@@ -102,92 +102,10 @@ describe('kernel exec - numeric instructions', () => {
 
     /**
      * Integer 64 bits
+     * 
+     * TODO: put these tests back in place!
```

Comment:
These tests are actually not feasible in JS. Might be worth investigating another way to test (stack serialization comparison, I think I saw ChakraCore doing it so)